File: src/main/java/cn/zjujri/workday/WorkdayApplication.java
package cn.zjujri.workday;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@EnableJpaRepositories
public class WorkdayApplication {

	public static void main(String[] args) {
		SpringApplication.run(WorkdayApplication.class, args);
	}

}

File: src/main/java/cn/zjujri/workday/WorkdayApplication.kt
package cn.zjujri.workday

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication
@EnableJpaRepositories
open class WorkdayApplication

fun main(args: Array<String>) {
    SpringApplication.run(WorkdayApplication::class.java, *args)
}
